pkg/pillar/cmd/zedagent/reportinfo.go

```diff
@@ -647,9 +646,11 @@ func PublishDeviceInfoToZedCloud(ctx *zedagentContext) {
 
 // PublishAppInstMetaDataToZedCloud is called when an appInst reports its Metadata to EVE.
 // AppInst metadata is relayed to the controller to be processed further.
-func PublishAppInstMetaDataToZedCloud(ctx *zedagentContext, appInstID string, appInstMetadata *types.AppInstMetaData,
-	metadataType types.AppInstMetaDataType, iteration int) {
-	log.Functionf("PublishAppInstMetaDataToZedCloud: appInstID: %v", appInstID)
+func PublishAppInstMetaDataToZedCloud(ctx *zedagentContext, pubsubKeyStr string,
+	appInstMetadata *types.AppInstMetaData, isDelete bool) {
```

Comment:
Didn't realize isDelete was already here. With that isDelete we no longer need the pubsubKeyStr argument.